### PR TITLE
style(kubernetes): updating the entire page

### PR DESCRIPTION
**Pull request by nbaenam**
Modifying the introduction page to Kubernetes:

- Updating the intro paragraph.
- Updating the [Get started: Install the Kubernetes integration](https://docs.newrelic.com/docs/kubernetes-pixie/kubernetes-integration/get-started/introduction-kubernetes-integration#install) section.
- Adding the info that is inside the _Navigate all your Kubernetes events_ inside the _Why it matters_ section. 
- Removing the _Navigate all your Kubernetes events_ title.

This is the ticket: [NR-64587](https://issues.newrelic.com/browse/NR-64587)

### CODE REVIEW

**Review by x8a — APPROVED** [review]
Looks great!

**Comment by nbaenam** [comment]
I'm waiting for a review from Drew Decker or Natalie Serrino.

**Pull request merged**
